### PR TITLE
styled the dropdown and added slightly more functionality

### PR DESCRIPTION
**Pull request by dannyscheiner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
   },
   "homepage": "https://github.com/oslabs-beta/nautilis#readme",
   "dependencies": {
+    "@types/yamljs": "^0.2.30",
```
Comment (michaelbayday) [inline package.json]: remove

File: package.json [hunk 2]
```diff
@@ -33,7 +34,8 @@
     "react": "^16.13.0",
     "react-bootstrap": "^1.0.0-beta.17",
     "react-dom": "^16.13.0",
-    "react-icons": "^3.9.0"
+    "react-icons": "^3.9.0",
+    "yamljs": "^0.3.0"
```
Comment (michaelbayday) [inline package.json]: remove

File: src/renderer/components/InfoDropdown.tsx [hunk 3]
```diff
@@ -12,169 +12,167 @@ import React from 'react';
 import { FaAngleDown } from 'react-icons/fa';
 import Accordion from 'react-bootstrap/Accordion';
 import Card from 'react-bootstrap/Card';
+// import YAML from 'yamljs';
```
Comment (michaelbayday) [inline src/renderer/components/InfoDropdown.tsx:15]: remove

File: src/renderer/components/InfoDropdown.tsx [hunk 4]
```diff
@@ -12,169 +12,167 @@ import React from 'react';
...
     });
   }
 
   return (
     <div className="info-dropdown">
-      <Accordion>
+      <h3>
+        {selectedContainer !== ''
+          ? selectedContainer[0].toUpperCase() + selectedContainer.slice(1)
+          : selectedContainer}{' '}
```
Comment (michaelbayday) [inline src/renderer/components/InfoDropdown.tsx]: i think you can remove {' '}

**Review by michaelbayday — COMMENTED** [review]
Nice work

**Review by jdnordy — COMMENTED** [review]
Good work dany!

**Review by jdnordy — APPROVED** [review]

**Pull request merged**